packages/ckeditor5-emoji/docs/features/emoji.md

```diff
@@ -1,24 +1,36 @@
 ---
 category: features
 menu-title: Emoji
-meta-title: Emoji | CKEditor 5 Documentation1
-modified_at: 2024-11-26
+meta-title: Emoji | CKEditor 5 Documentation
+modified_at: 2025-01-03
 ---
 
+{@snippet features/emoji-source}
+
 # Emoji
 
-The emoji feature lets you insert emojis from the editor toolbar, or directly on the go while writing the content.
+The emoji feature lets you insert emojis to the document from the editor toolbar, or directly on the go while writing the content.
+
+Since the number of emojis is very large, they have been conveniently divided into several categories for easier use, like "Smileys & Expressions" or "Objects". Besides, in order not to get lost, you may try to search for the emoji you are looking for using the search field.
 
 ## Demo
 
 Use the emoji toolbar button {@icon @ckeditor/ckeditor5-emoji/theme/icons/emoji.svg Emoji} in the editor below to open a panel with a table of selectable emojis.
 
+You may also trigger the emoji UI while writing the content when you type a pre-configured `:` marker followed by at least two letters of desired emoji name. Make sure to include at least one space before the marker, otherwise the emoji UI does not kick in. Then, you can either choose which emoji to insert from the suggestion list, or choose the last option &ndash; `Show all emoji...` &ndash; to open the full panel.
+
+Additionally, there is also an option to set the skin tone of the emojis next to the search field.
+
+The selected emoji category and the tone skin are remembered by the feature, so the next time you invoke the panel you can quickly insert a similar emoji as before.
+
 {@snippet features/emoji}
 
 <info-box info>
 	This demo presents a limited set of features. Visit the {@link examples/builds/full-featured-editor feature-rich editor example} to see more in action.
 </info-box>
 
+**Note:** Availability of emojis and their appearance depends on the operating system. If you want to standardize both the appearance and availability of emojis, please consider using an external font in your integration, such as [Noto Color Emoji](https://fonts.google.com/noto/specimen/Noto+Color+Emoji).
```

Comment:
Maybe we should wrap this in `<info-box warning>`?

Reply:
Done in https://github.com/ckeditor/ckeditor5/pull/17713/commits/a420c9ca4b2e296d5156a9ee5c7b74731fac734a.

![obraz](https://github.com/user-attachments/assets/cffb92ff-6f4e-4052-a7d5-8afde0750910)